releasenotes/notes/DogstatsdproxyFix-8d768c49b95d02b4.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    APM: Fix issue where dogstatsd proxy would not work when bind address was set to localhost on MacOS.
+    APM: Fix issue where setting bind_host to "::1" would break runtime metrics for the trace-agent.
```

Comment:
```suggestion
  - |
    APM: Fix issue where dogstatsd proxy would not work when bind address was set to localhost on MacOS.
  - |
    APM: Fix issue where setting bind_host to "::1" would break runtime metrics for the trace-agent.
```